torch/ao/quantization/backend_config/native.py

```diff
@@ -61,6 +62,14 @@
     is_dynamic=True,
 )
 
+# Needed for LayerNorm and f.layer_norm, since currently the kernel only supports float weights
+input_output_only_quint8_dtype_config = DTypeConfig(
+    input_dtype=torch.quint8,
+    output_dtype=torch.quint8,
+    weight_dtype=torch.float,
+    bias_dtype=torch.float,
```

Comment:
just thought about this again, looks like by default we assume the weight dtype is supported: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/prepare.py#L190 maybe the default should be float/not quantized?

Reply:
basically when weight_dtype is not specified in BackendConfig it is None, so the check for weight_dtype specified by QConfig will always pass right now, this doesn't sound correct I feel